### ISSUE TITLE
Tests fail on Termux

### ISSUE THREAD

**Issue by bpj** [jkeenan/file-copy-recursive-reduced#12]
TL;DR It seems “safe” to force install with cpanm…

I thoucght I’d just chime in and tell that lots of tests fail on [Termux] aka @termux, as far as I can tell all because paths aren’t the expected ones, which is not surprising given that Termux does absolutely crazy things to paths! Basically the root of the Termux file system (on my phone) is `/data/data/com.termux/files/` and that messes up a lot of things which rely on a standard \*nix file system. There is a command `termux-chroot` which makes the current session spoof a standard file system but that doesn’t help in case of the FCRR tests, possibly because of the things local::lib does to `PATH` etc. (I’m not sure.) FWIW Termux does define an environment variable `PREFIX` which expands to `/data/data/com.termux/files/usr` and might be useful.

  [Termux]: https://termux.dev/en

I’m at a loss how to fix this. However as I said at the top things currently do work after force installing with cpanm, although I don’t feel good about doing so! I’d say Termux is pretty niche, so it may be OK to settle for this at the moment. I thought I’d summit this report in the hope that Termux users looking for a fix find it rather than go nuts.

Sorry I can’t be of more help!
